content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -15,8 +15,7 @@ In particular, the LTS Release Line is based on slightly older releases to provi
 * *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin. 
 You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases. 
 * *Do not use versions no longer supported by the update center*, which is currently anything older than PLACEHOLDER_OLDEST_WEEKLY for weekly releases, and PLACEHOLDER_OLDEST_LTS for LTS releases.
-* *Prefer ".1" LTS releases over weekly versions and later releases within an LTS line* for greater compatibility.
-  LTS releases with the same baseline usually don't have differences that would matter to plugin developers, and if they do, they may well behave differently on supposedly compatible weekly releases published shortly after the LTS baseline.
+* *Prefer the last release within an LTS line over weekly versions and the 2.xxx.1 release within an LTS line*.
```

Comment:
not 100% happy with how this reads but I don't think .1 needs mentioning given you are recommending to use the last version
```suggestion
* *Prefer the last release within an LTS line over weekly versions.
```

Reply:
Probably needs to be massaged somehow. Maybe
```suggestion
* *Prefer an LTS version over weekly versions.*
  The last release in a historical line (often 2.xxx.3) may be more appropriate than the initial 2.xxx.1 release,
  and may be forced by dependencies, though you may prefer to stay with 2.xxx.1 while a line is active.
```